android/app/src/main/java/org/smartregister/fhircore/fragment/PatientListFragment.kt

```diff
@@ -144,4 +206,13 @@ class PatientListFragment : Fragment() {
     Toast.makeText(requireContext(), "Syncing...", Toast.LENGTH_LONG).show()
     patientListViewModel.syncUpload()
   }
+
+    override fun onSearchDone(isPatientFound: Boolean, patientLogicalId: String) {
```

Comment:
This callback flow can be replaced with `MVVM` flow, i.e 
```
    patientListViewModel.findPatient(patientID).observe(
      requireActivity(),
      {
        // excluded for brevity
      }
    )
```

Check comment on ViewModel

Reply:
https://github.com/opensrp/fhircore/pull/129#discussion_r637818602